Tests/SwiftParserTest/Expressions.swift

```diff
@@ -476,7 +476,7 @@ final class ExpressionTests: XCTestCase {
       "[(Int) -> #^DIAG^#throws Int]()",
       diagnostics: [
         // FIXME: We should suggest to move 'throws' in front of '->'
-        DiagnosticSpec(message: "Expected expression after '->' in array element"),
+        DiagnosticSpec(message: "Expected expression in array element"),
```

Comment:
Ignoring the throws problem, this not being something like "expected return type" is also an issue.